devtools-ci-index.yaml

```diff
@@ -4690,25 +4694,23 @@
             git_repo: fabric8-analytics-release-monitor
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: fabric8-analytics-release-monitor
-        - 'devtools-test-e2e-{test_url}-{test_suite}':
-            git_organization: fabric8io
-            git_repo: fabric8-test
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}-{feature_level}':
             test_url: prod-preview.openshift.io
             test_suite: planner
-            osio_creds: osio-ci-planner-002-preview
-            branch: 'master'
-            ee_test_start_time: '35 */2 * * *'
+            cluster: us-east-2a
+            feature_level: released
+            osio_creds: osio-ci-planner-002
```

Comment:
I do not see this secret in vault and some others from this PR
https://vault.devshift.net/ui/vault/secrets/devtools-osio-ci/list/e2e/

Reply:
Planner team's credentials are here https://vault.devshift.net/ui/vault/secrets/devtools-osio-ci/list/planner/

Reply:
@skryzhny you are right that they are mixed

Reply:
Fixed